Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.py

```diff
@@ -270,6 +270,90 @@ def purge_ediscovery_data(self, case_id, search_id, purge_type, purge_areas):
         body = {"purgeType": purge_type, "purgeAreas": purge_areas}
         return self.ms_client.http_request(method="POST", url_suffix=url, json_data=body, resp_type="response")
 
+    def run_estimate_statistics_request(self, case_id, search_id, statistics_options=None):
+        url = f"security/cases/ediscoveryCases/{case_id}/searches/{search_id}/estimateStatistics"
+        body = {}
+        if statistics_options:
+            # Handle lists or single values safely
+            if isinstance(statistics_options, list):
+                statistics_options = ",".join(statistics_options)
+            body["statisticsOptions"] = statistics_options
+
+        response = self.ms_client.http_request(
+            method="POST",
+            url_suffix=url,
+            json_data=body,
+            resp_type="response",
+            ok_codes=[202],
+        )
+
+        # Get the Location header which contains the location of the microsoft.graph.security.estimateStatisticsOperation
+        # that was created to handle the estimate.
+        location_url = response.headers.get("Location")
+        if not location_url:
+            raise DemistoException("Missing Location header in estimateStatistics response")
```

Comment:
whats the usecase this wont be available? is it a legitimate usecase?
```suggestion
            raise DemistoException("Estimate statistics is not available for this search_id")
```
i think this is a more user friendly error